model/src/main/kotlin/io/spine/validation/bound/BoundDetails.kt

```diff
@@ -28,37 +28,49 @@ package io.spine.validation.bound
 
 import io.spine.protodata.ast.Field
 import io.spine.protodata.ast.File
-import io.spine.protodata.ast.MessageType
 import io.spine.protodata.ast.PrimitiveType
 import io.spine.protodata.type.TypeSystem
 import io.spine.validation.RANGE
 
 /**
- * The context of validating a numeric option that constrains a field's value
- * with a minimum or maximum bound.
+ * A container for auxiliary data required to parse and validate
```

Comment:
The problem with this documentation is that it says "it's a bag of things related to a numeric bound".

Please try to give a more specific answer to "what it is" in this documentation. Then, it will be easier to give the name.

Just for the reference, my idea (prior to your PR) was `BoundDeclaration`, roughly meaning "the `bound` option declared <somewhere>"?